packages/matter.js/src/node/ServerNode.ts

```diff
@@ -189,6 +189,10 @@ export class ServerNode<T extends ServerNode.RootEndpoint = ServerNode.RootEndpo
         await this.#mutex;
     }
 
+    async advertiseNow() {
+        await this.act(agent => agent.get(NetworkServer).advertiseNow());
```

Comment:
This is why I was thinking a generic convenience method for invoking behavior methods might be nice... E.g. `await node.invoke.network.advertiseNow()`

Reply:
But still would need deep
Inside structure knowledge. Unthinkable exposing this on the node where you as dev also call "bringOnline" might be more natural?! Let's see which other Casios we find. I do not see too many more right now tbh.